src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+    <p className="Person__age">
+      {`I am ${person.age}`}
+    </p>
+    )}
+    <p className="Person__partner">
+      {(() => {
+        if (!person.isMarried) {
+          return 'I am not married';
+        }
+
+        if (person.isMarried && person.sex === 'm') {
+          return `${person.partnerName} is my wife`;
+        }
+
+        return `${person.partnerName} is my husband`;
+      })()}
```

Comment:
in jsx we trying to not use IIFE, so let's move all these conditions in some vars in our component body